tests/test_components/test_simulation.py

```diff
@@ -1034,22 +1034,21 @@ def test_mode_object_syms():
     )
 
 
-@pytest.mark.parametrize("size, log_level", [(1.0, None), (65, 30)])
-def test_warn_large_epsilon(caplog, size, log_level):
+@pytest.mark.parametrize("size, num_struct, log_level", [(1, 1, None), (50, 1, 30), (1, 11000, 30)])
```

Comment:
we'll need to make pass the log level as None, "warning", "warning" instead of None, 30, 30 respectively to pass with the recent changes.